galaxy_ng/app/pipelines.py

```diff
@@ -0,0 +1,47 @@
+from django.conf import settings
+from django.contrib.auth.models import Group
+
+
+def user_role(response, details, user=None, *args, **kwargs):
+    if not user:
+        return
+
+    roles = response.get(settings.KEYCLOAK_ROLE_TOKEN_CLAIM, [])
+    if not roles:
+        return
+
+    if not isinstance(roles, list):
+        return
+
+    is_admin = False
+    if settings.KEYCLOAK_ADMIN_ROLE in roles:
+        is_admin = True
+
+    user.is_staff = is_admin
+    user.is_admin = is_admin
+    user.is_superuser = is_admin
+    user.save()
+
+
+def user_group(response, details, user=None, *args, **kwargs):
+    if not user:
+        return
+
+    group_list = response.get(settings.KEYCLOAK_GROUP_TOKEN_CLAIM, [])
+    if not group_list:
+        return
+
+    if not isinstance(group_list, list):
```

Comment:
If I remove all groups from a user, the groups are never updated on the hub side.